bodhi/server/models.py

```diff
@@ -2380,16 +2380,23 @@ def num_admin_approvals(self):
     def test_cases(self):
         tests = set()
         for build in self.builds:
-            for test in build.package.test_cases:
+            test_cases = build.package.test_cases \
+                        .order_by(TestCase.id.asc())
```

Comment:
FWIW, the Bodhi style guide allows 100 column-wide code, so you can probably fit this on one line (I think, I didn't count ☺)